### ISSUE TITLE
Compatibility issue with unfix mui for pocohud 3

### ISSUE THREAD

**Issue by rockymoto517** [Crackdown-PD2/BigLobby3#43]
log shows this

10:37:25 PM Lua: Could not open file: mods/saves/autoexec.pc
10:37:25 PM FATAL ERROR:  (C:\projects\payday2-superblt\src\InitiateState.cpp:320) mods/Pocohud3 - Unfix MUI/hudmanagerpd2.lua:2: attempt to index global 'MUITeammate' (a nil value)
stack traceback:
	mods/Pocohud3 - Unfix MUI/hudmanagerpd2.lua:2: in main chunk
	[C]: in function 'dofile'
	mods/base/base.lua:160: in function 'RunHookFile'
	mods/base/base.lua:152: in function 'RunHookTable'
	mods/base/base.lua:182: in function 'OrigRequire'
	mods/BeardLib/Core.lua:361: in function 'require'
	[string "lib/managers/hudmanager.lua"]:1927: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:181: in function 'OrigRequire'
	mods/BeardLib/Core.lua:361: in function 'require'
	mods/More Weapon Stats/lua/menushotgunbase.lua:8: in main chunk
	...
	mods/BeardLib/Core.lua:361: in function 'require'
	[string "lib/setups/menusetup.lua"]:19: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:181: in function 'OrigRequire'
	mods/BeardLib/Core.lua:361: in function 'require'
	[string "lib/entry.lua"]:13: in main chunk
	[C]: in function 'require'
	mods/base/base.lua:181: in function 'OrigRequire'
	mods/BeardLib/Core.lua:361: in function 'require'
	[string "core/lib/coreentry.lua"]:19: in main chunk

basically makes you fail every time you try to join
